.github/workflows/ci-master.yml

```diff
@@ -12,9 +12,21 @@ jobs:
         target:
           - { name: Linux, os: ubuntu-latest, triple: x86_64-unknown-linux-gnu }
           - { name: macOS, os: macos-latest, triple: x86_64-apple-darwin }
-          - { name: Windows, os: windows-latest, triple: x86_64-pc-windows-msvc }
-          - { name: Windows (MinGW), os: windows-latest, triple: x86_64-pc-windows-gnu }
-          - { name: Windows (32-bit), os: windows-latest, triple: i686-pc-windows-msvc }
+          - {
+              name: Windows,
+              os: windows-latest,
+              triple: x86_64-pc-windows-msvc,
+            }
+          - {
+              name: Windows (MinGW),
+              os: windows-latest,
+              triple: x86_64-pc-windows-gnu,
+            }
+          - {
+              name: Windows (32-bit),
+              os: windows-latest,
+              triple: i686-pc-windows-msvc,
+            }
```

Comment:
if we want this to be formatted as i suspect was done with prettier then lets have it as normal yaml

```yaml
          - name: Windows
             os: windows-latest
             triple: x86_64-pc-windows-msvc
```

etc.

Reply:
Ah, it seems my editor does that automatically when replacing something. I don't have a favor and am going to revert.